### PR TITLE
*think [emote]

### PR DESCRIPTION
**Pull request by silicons**

### CODE REVIEW

**Comment by davidchan13** [comment]
>Not added to *help 

REEEEEE

**Comment by deathride58** [comment]
🤔 
@davidchan13 *help is automatically generated

**Comment by davidchan13** [comment]
@deathride58 If thats true why are none of the citadel specific emotes in the *help

File: code/modules/mob/living/emote.dm [hunk 1]
```diff
@@ -16,6 +16,13 @@
 	message_param = "bows to %t."
 	restraint_check = TRUE
 
+/datum/emote/living/think
+	key = "think"
+	key_third_person = "thinks"
+	message = "makes a thinking face."
```
Comment (nik707) [inline code/modules/mob/living/emote.dm:22]: "does a think"

File: code/modules/mob/living/emote.dm [hunk 2]
```diff
@@ -16,6 +16,13 @@
 	message_param = "bows to %t."
 	restraint_check = TRUE
 
+/datum/emote/living/think
+	key = "think"
+	key_third_person = "thinks"
+	message = "makes a thinking face."
+	message_param = "makes a thinking face at %t."
```
Comment (nik707) [inline code/modules/mob/living/emote.dm:23]: ditto

**Review by nik707 — CHANGES_REQUESTED** [review]
get it together

**Pull request closed without merging**